Mathlib/CategoryTheory/Limits/Shapes/Types.lean

```diff
@@ -711,12 +711,206 @@ end Pullback
 
 section Pushout
 
--- #synth HasPushouts.{u} (Type u)
-instance : HasPushouts.{u} (Type u) :=
-  -- FIXME does not work via `inferInstance` despite `#synth HasPushouts.{u} (Type u)` succeeding.
-  -- https://github.com/leanprover-community/mathlib4/issues/5752
-  -- inferInstance
-  hasPushouts_of_hasWidePushouts.{u} (Type u)
+variable {S X₁ X₂ : Type u} (f : S ⟶ X₁) (g : S ⟶ X₂)
+
+/-- The pushout of two maps `f : S ⟶ X₁` and `g : S ⟶ X₂` is the quotient
+by the equivalence relation on `X₁ ⊕ X₂` generated by this relation. -/
+inductive Pushout.Rel (f : S ⟶ X₁) (g : S ⟶ X₂) : X₁ ⊕ X₂ → X₁ ⊕ X₂ → Prop
+  | inl_inr (s : S) : Pushout.Rel f g (Sum.inl (f s)) (Sum.inr (g s))
+
+/-- Construction of the pushout in the category of types, as a quotient of `X₁ ⊕ X₂`. -/
+def Pushout : Type u := _root_.Quot (Pushout.Rel f g)
+
+/-- In case `f : S ⟶ X₁` is a monomorphism, this relation is the equivalence relation
+generated by `Pushout.Rel f g`. -/
+inductive Pushout.Rel' : Sum X₁ X₂ → Sum X₁ X₂ → Prop
```

Comment:
Please use notation `X ⊕ Y` for `Sum X Y`. Mathlib rarely uses it because Mathport deleted it all over the library.